### PR TITLE
Setup a behat yml automatically

### PR DESCRIPTION
**Pull request by dawehner**
It would be nice to setup a behat.local.yml automatically which contains the required environment specific settings. This change just replaces the placeholders in the file, so you can add details as you like

### CODE REVIEW

File: src/Stacks/Drupal.php [hunk 1]
```diff
@@ -91,28 +92,41 @@ protected function ensureLocalSettings() {
...
         file_put_contents($target_local_settings, $localSettings);
 
         // Relink if missing.
         if (!$this->fs->exists(Platform::webDir() . '/sites/default/settings.local.php')) {
             $this->fs->symlink($this->getRelativeLinkToShared() . '/settings.local.php', Platform::webDir() . '/sites/default/settings.local.php');
         }
-    }
 
+      if (file_exists(Platform::rootDir() . '/tests/behat.local.yml.dist')) {
+          $behat_local_yml = file_get_contents(Platform::rootDir() . '/tests/behat.local.yml.dist');
```
Comment (alexpott) [inline src/Stacks/Drupal.php:125]: Doing a replace on the actual file will be a pain for keeping in .git

File: src/Stacks/Drupal.php [hunk 2]
```diff
@@ -91,28 +92,41 @@ protected function ensureLocalSettings() {
...
+          '{{ salt }}' => hash('sha256', serialize($_SERVER)),
+          '{{ container_name }}' => $this->containerName,
+          '{{ redis_container_name }}' => $this->redisContainerName,
+          '{{ solr_container_name }}' => $this->solrContainerName,
+          '{{ project_domain }}' => $this->projectName . '.' . $this->projectTld,
+          '{{ external_project_domain }}' => 'http://localhost/' . trim(shell_exec($detect_port_command)),
+          '{{ mysql_user }}' => Mysql::getMysqlUser(),
+          '{{ mysql_password }}' => Mysql::getMysqlPassword(),
+          '{{ project_root }}' => Platform::rootDir(),
+        ];
```
Comment (alexpott) [inline src/Stacks/Drupal.php:107]: Nice - been mean to do this myself.

File: src/Stacks/Drupal.php (hunk 1 above)
Comment (dawehner) [reply src/Stacks/Drupal.php:125]: Well, it copies the behat.local.yml.dist to behat.local.yml

Comment (dawehner) [reply src/Stacks/Drupal.php:125]: Which is in the .gitignore

Comment (alexpott) [reply src/Stacks/Drupal.php:125]: I should have sharper eyes :)

**Comment by dawehner** [comment]
Nice!

**Pull request merged**